Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -121,6 +121,12 @@ public static async Task LaunchStoreForReviewAsync()
         /// </summary>
         public PackageVersion FirstVersionInstalled { get; }
 
+        /// <summary>
+        /// Gets the previous version of the app that was installed.
+        /// This will be the current version if a previous version of the app was installed before accessing this property.
```

Comment:
Nit: this is not actually correct though, right? As in, the property is initialized along with the persisted value in the app settings when the `SystemInformation` instance is initialized (the relative code is triggered from the constructor), so it gets executed even if this property is not actually used. The phrasing should be changed to indicate that the current version might be returned if the previous version was installed before using `SystemInformation` in general, and not necessarily this specific property.

Reply:
Hi @Sergio0694 good catch, to be fair I just reused the `FirstVersionInstalled` wording, should be fixed also this one?